config/app.json

```diff
@@ -2,7 +2,7 @@
   "mkimage": {
     "listen": 3020,
     "access_log": "",
-    "cache_dir": "/vol0/imagecache",
+    "cache_dir": "/tmp/imagecache",
```

Comment:
this seems fine to me as we use our own manifest configs when warmed on the prod cluster